data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.      df = dataset.load_reference_samples(org_name=ORG_NAME,
</pre><p>Did you see Ben and my comments earlier? TL;DR: It's best to use <code>eeddf</code> directly instead of the dataset API. If you want to continue using the dataset API, make sure to intialize <code>eeddf</code> manually per the code example in chat.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I don't remember Ben said anything about preferring eedf. I mentioned that dataset does the same and there weren't contentious comments.

Reply:
Ben:
"""
though technically I would prefer you switch to use:import eeddf
importlib.reload(eeddf)  <==auto reloads dependent modules so you dont have to

eeddf.load_reference_samples(...)  <== same api as dataset, but in a unified module

but all the details nicholas has are correct. see https://github.com/tnc-br/ddf_common/blob/main/ddf_ee_api.ipynb  for example
"""

So it's better if we explicitly initialize eeddf and use eeddf.load_reference_samples instead of the `dataset` library. That's the chat I was referring to.